### PR TITLE
Add shutdown-duration-delay to openshift-apiserver

### PR DESCRIPTION
**Pull request by jonesbr17**
This is meant to mirror changes made to core OpenShift via https://github.com/openshift/cluster-openshift-apiserver-operator/commit/cad9746b62abf3b3230592d45f7f60bcecc96dac

### CODE REVIEW

File: assets/openshift-apiserver/openshift-apiserver-deployment.yaml [hunk 1]
```diff
@@ -151,6 +151,16 @@ spec:
           timeoutSeconds: 1
           failureThreshold: 3
           successThreshold: 1
+        startupProbe:
```
Comment (rtheis) [inline assets/openshift-apiserver/openshift-apiserver-deployment.yaml:154]: We don't do need a startup probe for this.

**Review by rtheis — CHANGES_REQUESTED** [review]

File: assets/openshift-apiserver/openshift-apiserver-deployment.yaml (hunk 1 above)
Comment (jonesbr17) [reply assets/openshift-apiserver/openshift-apiserver-deployment.yaml]: Ok, I saw this added upstream in the same commit as the shutdown-duration-delay flag, so I included it here as well. I removed it in the latest commit.

**Review by rtheis — APPROVED** [review]
/lgtm
/approved

**Pull request merged**